src/content/docs/apm/agents/go-agent/configuration/go-agent-configuration.mdx

```diff
@@ -3070,3 +3070,355 @@ Normally all of the options you set as part of your agent configuration are repo
   </Collapser>
 
 </CollapserGroup>
+
+## Security Agent
+
+[New Relic Security agent interactive applications security testing](https://docs.newrelic.com/docs/iast/introduction/) (IAST) tests your applications for any exploitable vulnerability by replaying the generated HTTP request with vulnerable payloads. Similar to the go agent config, the security agent is configured through option functions that are passed to the Init function.
+It can alternatively be configured by environment variables, or from a YAML file as well. Options set using config options will take prescedence over environment or YAML configurations.
```

Comment:
@iamemilio Wanted to clarify: I thought that YAML configs ARE the config file, and environment variables are separate. So there are two options for setting agent configs. This paragraph makes it seem like there are three:

1. Config file
2. Enviro
3. YAML

Can you clarify this?